Formula/lfe.rb

```diff
@@ -4,6 +4,7 @@ class Lfe < Formula
   url "https://github.com/lfe/lfe/archive/v2.0.tar.gz"
   sha256 "373ad033bb74679766ed7414bfd6d69fbf6dd0a2337019eb117840384b9fada0"
   license "Apache-2.0"
+  revision 1
```

Comment:
I'm thinking maybe could skip the `revision`. Not a strong opinion though.

Reply:
Yeah, wasn't sure. But since this apparently massively impacts users of FLE I opted to revision